src/contracts/exchange/NiftyswapExchange20.sol

```diff
@@ -742,6 +857,29 @@ contract NiftyswapExchange20 is ReentrancyGuard, ERC1155MintBurn, INiftyswapExch
     return factory;
   }
 
+  /**
+   * @return Global royalty fee % if not supporting ERC-2981
+   */
+  function getRoyaltyFee() override external view returns (uint256) {
```

Comment:
Aren't these redundant with `getRoyaltyInfo`? It's weird because they return `0` if `ERC-2981` is enabled, that may give the impression that a pool has no royalty fees even when it has

Reply:
yeah good point, i'll get rid of this one and the recipient one